### DISCUSSION TITLE
News Cache [2024-06-17]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#7226] [Announcements]
### 🚢  Ships

- [Change the default position of your Vercel Toolbar](https://vercel.com/changelog/reposition-the-vercel-toolbar)
- [Azure AI Provider for the Vercel AI SDK now available](https://vercel.com/changelog/azure-ai-provider-for-the-vercel-ai-sdk-now-available)
- [HTML element attribution in Speed Insights](https://vercel.com/changelog/html-element-attribution-in-speed-insights)
- [CSV Export in Web Analytics](https://vercel.com/changelog/csv-export-in-web-analytics)

### 📚 Info

- @pawlean just joined the Vercel Community team, and she’ll be joining our discussions here very soon!
- [How we improved Vercel cold starts](https://www.youtube.com/watch?v=53zrROpR3ZY)
- [DevTools for every Vercel deployment](https://www.youtube.com/watch?v=DAY8ATPBK2k)

### ℹ️ Reminders

- [Account-owned domains now visible in team-scope domains tab](https://vercel.com/changelog/account-owned-domains-now-visible-in-team-scope-domains-tab)

**Comment by pawlean** [comment]
Excited to be here [[1](https://x.com/paulienuh/status/1801313673493025282)][[2](https://www.linkedin.com/posts/pnarvas_nextjs-vercelship-activity-7207079275898761216-WkXQ?utm_source=share&utm_medium=member_desktop)]! 🥳

[a comment by an automated account is not shown]
